src/Symfony/Bundle/WebProfilerBundle/composer.json

```diff
@@ -17,7 +17,7 @@
     ],
     "require": {
         "php": "^5.5.9|>=7.0.8",
-        "symfony/http-kernel": "~3.3|~4.0",
+        "symfony/http-kernel": "~3.4.25|~4.1.12|~4.2.6",
```

Comment:
`~3.4.25|^4.2.6` (4.1 doesn't receive bugfixes anymore and we also do not want to exclude +4.3)

Reply:
Ah right, `~` vs `^`, missed that one!